portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,76 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Kristen's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Kristen's Portfolio</h1>
+      <div class="tab">
+        <button class="tablinks active" onclick="openTab(event,'home')">Home</button>
+        <button class="tablinks" onclick="openTab(event,'about-me')">About Me</button>
+        <button class="tablinks" onclick="openTab(event,'my-projects')">My Projects</button>
+        <button class="tablinks" onclick="openTab(event,'fun-facts')">Fun Facts</button>
+        <button class="tablinks" onclick="openTab(event,'travel-blog')">Travel Blog</button>
+        <button class="tablinks" onclick="openTab(event,'contact-me')">Contact Me</button>
+      </div>
+        <div id="home" class="tabcontent">
```

Comment:
Looks like some of your indentation in this file is a bit off. Try to make sure that the closing tab always matches the opening tag. Anything inside of a tag should be more indented than its container. Sibling elements should have the same indentation

Reply:
Resolved